neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jStatement.java

```diff
@@ -271,63 +274,50 @@ protected void checkClosed() throws SQLException {
 	}
 
 	@Override public int getMaxFieldSize() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
+		return 0;
 	}
 
-	@Override public void setMaxFieldSize(int max) throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
+	@Override public void setMaxFieldSize(int max) throws SQLException {} // do nothing
 
-	@Override public void setEscapeProcessing(boolean enable) throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
+	@Override public void setEscapeProcessing(boolean enable) throws SQLException {} // do nothing
 
-	@Override public void setCursorName(String name) throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
+	@Override public void setCursorName(String name) throws SQLException {} // do nothing
 
-	@Override public void setFetchDirection(int direction) throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
+	@Override public void setFetchDirection(int direction) throws SQLException {}
 
 	@Override public int getFetchDirection() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
+		return ResultSet.FETCH_FORWARD;
 	}
 
 	@Override public int getFetchSize() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
+		return 0;
 	}
 
 	@Override public boolean getMoreResults(int current) throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
+		return getMoreResults();
 	}
 
 	@Override public ResultSet getGeneratedKeys() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
+		// TODO should at least return the <id>?
+		return ListNeo4jResultSet.newInstance(false, Collections.<List<Object>>emptyList(), Collections.<String>emptyList());
 	}
 
-	@Override public void cancel() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
+	@Override public void cancel() throws SQLException {} // do nothing
 
-	@Override public void setPoolable(boolean poolable) throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
+	@Override public void setPoolable(boolean poolable) throws SQLException {} // do nothing
 
 	@Override public boolean isPoolable() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
-	}
-
-	@Override public int[] executeBatch() throws SQLException {
-		throw ExceptionBuilder.buildUnsupportedOperationException();
```

Comment:
that's an interesting one that we should look into at some point, can you add an issue